doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -132,27 +97,16 @@ Model training
 
 The model training process is relatively simple and involves collating a series of
 forecast-observation pairs with the associated feature variables into a single pandas
-dataframe.
-
-Using this dataframe, a series of lightGBM binary-classification models are trained against
-the truth values for exceedances of the forecast error relative to a series of error-threshold
-values, resulting in one model per error threshold.
-
-As the calibration method works on a per realization basis, the errors used in training
-must be representative of the underlying systematic model biases for a single ensemble
-member rather than those of the ensemble as a whole. This requires the use of a single
-representative ensemble forecast member in training. It also requires the underlying
-weather type associated with the forecast and observation in the forecast-observation
-pair be as consistent as possible.
-
-For the forecast values used in training, we use the control at the shortest available
-lead-times that distinctly represent of the underlying weather types. Specifically, we
-take the earliest 24-hour period for daily accumulations, and all lead-times over the
-earliest 24-hour period for sub-daily accumulations to capture the full cycle of diurnal
-variations.
+dataframe. In general, each ensemble member yields a separate row of the dataframe (although for 
+reasons of computational efficiency it may be desirable to only use a subset of members, for example 
+by using only the control realization in each ensemble). 
+Each model predicts the probability that rainfall will exceed a particular threshold, 
+so the output is expected to be a number between 0 and 1. The Improver code will cap model predictions outside 
+this range so they lie between 0 and 1. (This is necessary because we are currently training using the mean 
+squared error loss, there are occasional cases where the LightGBM model predicts vlaues that are slightly more than 1 
+or less than 0.)
```

Comment:
I think this is also a good update, but I might suggest rewording slightly:
```suggestion
As each model predicts the probability that rainfall will exceed a particular threshold, 
the output is expected to be a number between 0 and 1. However, on occasion the GBDT models may
predict values slightly outside of this range and so the model predictions are capped to back onto this
interval; such values are possible due to the current choice of loss function used in training, namely the
mean squared loss function which is akin to Brier score when working in probability space.
```

Reply:
done